scaffolder-templates/advanced-workflow/template.yaml

```diff
@@ -162,7 +162,74 @@ spec:
                 - persistencePSQLServiceNamespace
                 - persistencePSQLServicePort
                 - persistencePSQLDatabaseName
+    - title: Create Quay Repository
+      required:
+        - quayCreateNewRepo
+        - CI
+      properties:
+        quayCreateNewRepo:
+          title: Create a new Quay repository with the provided name? (You must select a CI/CD Method in the Build Environment)
+          type: string
+          default: "Yes"
+          enum:
+            - "Yes"
+            - "No"
+      dependencies:
+        quayCreateNewRepo:
+          oneOf:
+            - properties:
+                quayCreateNewRepo:
+                  const: "No"
+            - properties:
+                quayCreateNewRepo:
+                  const: "Yes"
+                token:
+                  title: Quay AuthToken
+                  type: string
+                  description: Bearer token used for authorization
+                  ui:widget: password
+                visibility:
+                  title: New quay Repository Visiblity
+                  type: string
+                  description: Visibility setting for the created repository, either public or private
+                  ui:widget: select
+                  enum: ["public", "private"]
+                repoKind:
+                  title: New Quay Repository Kind
+                  type: string
+                  description: The created repository kind either image, application or none
+                  ui:widget: select
+                  enum: ["image", "application"]
+                newRepoDescription:
+                  title: New Quay Repository Description
+                  type: string
+                  description: The repository desription
+                baseUrl:
+                  title: Quay Base URL
+                  type: string
+                  description: URL of your quay instance, set to "https://quay.io" by default
+                quayNamespace:
+                  title: Quay Namespace
+                  type: string
+                  description: Quay Namespace. Defaults to the namespace of the token.
```

Comment:
some descriptions ends with a dot, some aren't. let's go with a common practice for all.